docs/Examples2/Box.example.purs

```diff
@@ -1,12 +1,13 @@
 module Lumi.Components2.Examples.Box where
 
 import Prelude
+
```

Comment:
These spaces after `import Prelude` were automatically added by `psc-ide` when performing the auto-import command.

Reply:
And `purty` always removes them.. I wish they agreed on this formatting

Reply:
Oh no! 😓  I'll remove it then, since `purty` runs for you on every file save.

Reply:
It's not a big deal, whichever you prefer :)